TheMovieDatabase/ViewControllers/DetailedMovie/View/DetailedMovieViewController.swift

```diff
@@ -16,8 +16,10 @@ final class DetailedMovieViewController: UIViewController {
     static let identifier = String(describing: DetailedMovieViewController.self)
 
     var movieId: Int?
+    var movie: Movie?
     private let extractor = LinkExtractor()
-    private let service = MoviesLoadingService()
+    private let moviesLoadingService = MoviesLoadingService()
+    private let profileService = ProfileService()
```

Comment:
You use profile service there only for `saveFavorite`? Maybe better to put this method in MoviesLoadingService as I said?
Btw: now you can't call it MoviesLoadingService 'cause you have save methods there, just rename to MoviesService (and LoadingService looks ugly for me)

Reply:
probably you're right